src/GitVersion.Core/Configuration/ConfigurationProvider.cs

```diff
@@ -54,8 +55,24 @@ public void Init(string workingDirectory)
     {
         var overrideConfigurationFromFile = this.configFileLocator.ReadOverrideConfiguration(workingDirectory);
 
-        var configurationBuilder = GitFlowConfigurationBuilder.New;
-        foreach (var item in new[] { overrideConfigurationFromFile, overrideConfiguration })
+        var workflow = GetWorkflow(overrideConfiguration, overrideConfigurationFromFile);
+
+        Dictionary<object, object?>? overrideConfigurationFromWorkflow = null;
+
+        if (!string.IsNullOrEmpty(workflow))
+        {
+            var workflowResource = (byte[]?)Resources.ResourceManager.GetObject(workflow, Resources.Culture);
```

Comment:
I think we need to have a similar approach we have in the output - embed the resources instead of resx files -> check the https://github.com/GitTools/GitVersion/blob/main/src/GitVersion.Output/TemplateManager.cs

Reply:
done